editor/src/components/canvas/controls/classname-select.tsx

```diff
@@ -66,12 +157,160 @@ export const ClassNameSelect: React.FunctionComponent = betterReactMemo('ClassNa
+      maxWidth: 0,
+    }),
+
+    multiValue: () => {
+      return {
+        cursor: 'pointer',
+        display: 'flex',
+        alignItems: 'center',
+        height: 18,
+        backgroundColor: '#191818',
+      }
+    },
+    multiValueLabel: () => ({
+      fontSize: 10,
+      padding: '2px 4px',
+      color: 'white',
+    }),
+    multiValueRemove: (styles: React.CSSProperties, { data }) => ({
+      width: 11,
+      display: 'flex',
+      paddingTop: 2,
+      opacity: 0.4,
+      color: data.color,
+      ':hover': {
+        opacity: 1,
+        backgroundColor: data.color,
+        color: 'white',
+      },
+    }),
+    input: () => {
+      return {
+        fontSize: 11,
+        color: theme.fg1.value,
+        letterSpacing: 0.3,
+        background: 'transparent',
+        display: 'flex',
+        alignItems: 'center',
+      }
+    },
+    indicatorsContainer: (styles) => ({
+      ...styles,
+      height: 20,
+    }),
+    option: (styles: React.CSSProperties, { data, isDisabled, isFocused, isSelected }) => {
+      // a single entry in the options list
+      const categories = data?.categories ?? []
+
+      let optionColor = chroma('black')
+      if (categories.length === 1) {
+        optionColor = chroma(theme.primary.value)
+      }
+
+      let backgroundColor: string | undefined = undefined
+      if (isFocused) {
+        backgroundColor = optionColor.alpha(0.1).css()
+      } else if (isSelected) {
+        backgroundColor = optionColor.css()
+      } else if (isDisabled) {
+        backgroundColor = undefined
+      }
+
+      let activeBackgroundColor =
+        !isDisabled && (isSelected ? data.color : optionColor.alpha(0.3).css())
+
+      let color: string = optionColor.css()
+      if (isSelected) {
+        if (chroma.contrast(optionColor, 'white') > 2) {
+          color = 'white'
+        } else {
+          color = 'black'
+        }
+      } else if (isDisabled) {
+        color = '#ccc'
+      }
+
+      return {
+        minHeight: 27,
+        display: 'flex',
+        alignItems: 'center',
+        paddingLeft: 8,
+        paddingRight: 8,
+        backgroundColor: backgroundColor,
+        color: color,
+        cursor: isDisabled ? 'not-allowed' : 'default',
+
+        ':active': {
+          ...(styles as any)[':active'],
+          backgroundColor: activeBackgroundColor,
+        },
+      }
+    },
+  }
+
   return (
-    <WindowedSelect
-      isMulti={true}
-      options={TailWindOptions}
-      onChange={onSubmitValue}
-      value={selectedValues}
-    />
+    <div
+      css={{
+        backgroundColor: theme.bg1.value,
+        height: 22,
+        borderRadius: 3,
+        position: 'relative',
+        flexGrow: 1,
+        display: 'flex',
+        alignItems: 'center',
+        '&:focus-within': { boxShadow: `0px 0px 0px 1px ${theme.primary.value}` },
+      }}
+    >
+      <WindowedSelect
+        filterOption={createFilter({ ignoreAccents: false })}
```

Comment:
The `createFilter` call should probably be wrapped in a `React.useMemo` (or `React.useCallback`?)